.github/workflows/deploy-to-server.yml

```diff
@@ -0,0 +1,31 @@
+name: Automated deploy to server
+
+on:
+  workflow_dispatch:
+  push:
+    branches: 
+      - 'master'
+      - 'feature/upload-to-server'
+
+jobs:
+  deploy:
+    runs-on : ubuntu-latest
+    steps:
+      - name: multiple command
+        uses: appleboy/ssh-action@master
+        with:
+          command_timeout: 200m
+          host: '79.101.34.218'
```

Comment:
We shouldn't expose this as is we should mask it as git secret

Reply:
Taken care of